concepts/comparison/about.md

```diff
@@ -1,44 +1,87 @@
 # About
 
-TODO: this is a stub that needs more information
+## Comparing Numbers
 
-## Comparing numbers
+In JavaScript numbers can be compared using the following relational and equality operators.
 
-Numbers are considered equal if they have the same value.
+| Comparison             | Operator  |
+| ---------------------- | --------- |
+| Greater than           | `a > b`   |
+| Greater than or equals | `a >= b`  |
+| Less than              | `a < b`   |
+| Less than or equals    | `a <= b`  |
+| (Strict) Equals        | `a === b` |
+| Not (strict) equals    | `a !== b` |
+
+The result of the comparison is always a boolean value, so either `true` or `false`.
 
 ```javascript
-1 == 1.0;
+1 < 3;
 // => true
 
+2 !== 2;
+// => false
+
 1 === 1.0;
 // => true
-// Remember, all numbers are floating-points, so this is different syntax for
+// All numbers are floating-points, so this is different syntax for
 // the exact same value.
+```
 
-1 === 1n;
+## Comparing Strings
+
+In JavaScript the comparison operators above can also be used to compare strings. In that case a dictionary (lexicographical) order is applied. You can find a list of the exact order of all the characters [here][utf-16-list].
+
+```javascript
+'Apple' > 'Pear';
 // => false
-// Strictly checking a number against a bigint will always result in false.
 
-1 == 1n;
+'a' < 'above';
 // => true
-// A number is equal to a bigint if they represent the same value.
 
-1.0 == 1n;
+'a' === 'A';
+// => false
+```
+
+You need to be careful when you compare two variables that contain numeric values but are of type string. Due to the dictionary order the result will not be the same as comparing values of type number.
```

Comment:
Great example, and common pitfall imo.

Reply:
Indeed, excellent content! Once we support the new info blocks (see https://github.com/exercism/docs/pull/84), this might be a perfect candidate.